addons/account_edi_ubl_cii_tests/__manifest__.py

```diff
@@ -0,0 +1,19 @@
+# -*- coding: utf-8 -*-
+{
+    'name': "Testing the Import/Export invoices with UBL/CII",
+    'version': '1.0',
+    'category': 'Accounting/Accounting',
+    'depends': [
+        'l10n_generic_coa',
+        'account_edi_ubl_cii',
+        'l10n_fr',
```

Comment:
I don't think you want that. Here you are saying that to install this module, it should install l10n_fr, l10n_be, l10n_de, l10n_nl to be able to use your EDI. Probably what you wanted to do is to install your EDI when you are installing a localisation.

Sadly, as you are in stable, you cannot modify depends on existing module, so the customers in 15.0 will have to install you edi manually.

You'll be able to put the correct dependencies only in master.

FYI: @odoo/rd-runbot

Reply:
ok, but at least name it starting with "l10n_..." to avoid installing localisations on runbot, which slows the installation of databases on runbot.